arelle/FileSource.py

```diff
@@ -49,7 +49,7 @@ def openFileSource(
         return filesource
     else:
         if cntlr and base:
-            filename = cntlr.webCache.normalizeUrl(filename, base=base) # type: ignore[no-untyped-call]
+            filename = cntlr.webCache.normalizeUrl(filename, base=base)
```

Comment:
arbitrary line

@gmongelli thanks for doing this. There are merge conflicts that need to be resolved first, but we'd love to get this merged. Thanks again! :)

Reply:
The conflicts have been resolved!